libraries/core-react/CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.7.0] - 2020-11-26
+
+### Added ✨
+
+- Types, as part of the [Typescript Milestone](https://github.com/equinor/design-system/milestone/7?closed=1)
+- Created `NativeSelect` component ([#509](https://github.com/equinor/design-system/issues/509))
+- Added `Table` caption ([#621](https://github.com/equinor/design-system/issues/621))
+- Added `isExpanded` prop to be able to control `Accordion` externally ([#677](https://github.com/equinor/design-system/issues/677))
```

Comment:
No need to use “Added” since it’s already in the heading. Would also remove “Created”.